File: bench/Reference/process_spawn.js
const { execSync } = require('child_process');
const command = process.platform === 'win32'
    ? 'cmd /c for /l %i in (1,1,10000) do @echo %i'
    : 'seq 1 10000';
const output = execSync(command, { encoding: 'utf8' });
const normalized = output.replace(/\r\n/g, '\n').replace(/\r/g, '\n');
console.log(normalized.replace(/\n$/, '').length);

File: bench/Reference/process_spawn.py
import subprocess

if __name__ == "__main__":
    if __import__("os").name == "nt":
        cmd = ["cmd", "/c", "for /l %i in (1,1,10000) do @echo %i"]
    else:
        cmd = ["seq", "1", "10000"]
    result = subprocess.run(cmd, capture_output=True, text=True, check=True)
    normalized = result.stdout.replace("\r\n", "\n").replace("\r", "\n")
    print(len(normalized.rstrip("\n")))
